internal/models/chequebook.go

```diff
@@ -0,0 +1,51 @@
+package models
+
+import (
+	"fmt"
+
+	"github.com/chain4travel/camino-messenger-bot/pkg/cheques"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type ChequeTxStatus uint8
+
+const (
+	ChequeTxStatusUnknown ChequeTxStatus = iota
+	ChequeTxStatusProcessing
+	ChequeTxStatusAccepted
+	ChequeTxStatusRejected
+)
+
+func (c ChequeTxStatus) String() string {
+	switch c {
+	case ChequeTxStatusUnknown:
+		return "unknown"
+	case ChequeTxStatusProcessing:
+		return "processing"
```

Comment:
There is no transaction state as "processing". I think we should change this to "pending"? Wdyt?